src/uu/install/src/install.rs

```diff
@@ -583,6 +583,15 @@ fn standard(mut paths: Vec<String>, b: &Behavior) -> UResult<()> {
                 }
             }
         }
+
+        // in case we have -t and -D pointing to a file instead of a directory
+        if let Some(target_dir) = &b.target_dir {
+            let p = Path::new(target_dir);
+            if !p.exists() || !p.is_dir() {
+                show_error!("failed to access '{}': Not a directory", p.display());
+                return Err(1.into());
```

Comment:
Why don't you return an `InstallError` here like in the other error cases of this function?

Reply:
because it followed what was done above in the file and forgot to move it here :)
I will fix it, thanks